team_comps.py

```diff
@@ -5,35 +5,47 @@
 #         print("KEY = " + key)
 #     return tier_S
 
-def get_data():
-    pprint(data)
-    return data
 
-def build_data(input):
-    user_data.append(input)
-    print(user_data)
+def get_team_data():
+    pprint(team_data)
+    return team_data
+
+
+def add_champ(champ_name):
+    user_team.append(champ_name)
+    print(user_team)
```

Comment:
Add error catching here that prevents the user from entering a blank string or a champion that doesn't exist in the data set. Ex: "", "dinx", etc